lms/templates/edxnotes/note-item.underscore

```diff
@@ -17,7 +17,12 @@
     <ol class="note-comments" role="region" aria-label="<%- gettext('Note') %>">
         <li class="note-comment">
             <p class="note-comment-title"><%- gettext("You commented...") %></p>
-            <p class="note-comment-p"><%= text %></p>
+            <p class="note-comment-p">
+                <%= interpolate_text(_.escape(text), {
+                    highlighted_notes_start: '<span class="note-highlight">',
```

Comment:
This was done as discussed, but I don't like that the tag definition (name + delimiters) is in Python, but the tag replacement is in Underscore.

I see 2 options:
1. Leave the code as is, except rename tags to `elasticsearch_highlight_start` and `elasticsearch_highligh_end`.
  * The rename makes it more clear where the tags are coming from, but I don't like this.  
2. Do the escaping (`markupsafe.escape()`) and tag replacement in the Python code.
  * The downside of this is that the `<span>` is moved from the template back into Python
  * If this is done, you should rename `text` to `text_html`, so it will be clear why you need to use `<%=` again.

I vote for #2.

@andy-armstrong Thoughts on this?